src/cryptoadvance/specter/managers/config_manager.py

```diff
@@ -143,11 +143,7 @@ def update_explorer(self, explorer_id, explorer_data, user, chain):
                 # make sure the urls end with a "/"
                 explorer_data["url"] += "/"
         else:
-            chain_name = (
-                ""
-                if (chain == "main" or chain == "regtest")
-                else ("signet/" if self.chain == "signet" else "testnet/")
-            )
+            chain_name = "" if (chain == "main" or chain == "regtest") else "testnet/"
```

Comment:
Why getting rid of the signet prefix (signet/)?

Reply:
I couldn't see the value for a signet-explorer but the actual problem was the `self.chain`. Fixed.